test-runner/src/App.css

```diff
@@ -44,4 +42,10 @@ button:focus {
 .code-window-titles {
   font-size: 15px;
   margin: 10px;
-}
\ No newline at end of file
+}
+
+.editor-div {
```

Comment:
This setup allows each code mirror instance on the page to fill an equal amount of space vertically! Before this we were doing some less than idea layout management with things like `calc(50vh - 100px)`